src/System.Private.CoreLib/shared/System/Globalization/TextInfo.cs

```diff
@@ -13,19 +13,29 @@
 ////////////////////////////////////////////////////////////////////////////
 
 using System.Diagnostics;
+using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
 using System.Runtime.Serialization;
 using System.Text;
+using Internal.Runtime.CompilerServices;
+
+#if BIT64
+using nuint = System.UInt64;
+using nint = System.Int64;
+#else // BIT64
+using nuint = System.UInt32;
+using nint = System.Int32;
+#endif // BIT64
 
 namespace System.Globalization
 {
     public partial class TextInfo : ICloneable, IDeserializationCallback
     {
         private enum Tristate : byte
         {
-            NotInitialized,
-            True,
-            False,
+            True = 1,
+            False = 0,
+            NotInitialized = 0x80,
```

Comment:
Do you really need to change NonInitalized to be 0x80? It is kind of nice for NonInitalized to be 0 for flags like these.